audit/projects/k8s-artifacts-prod/services/bigquery/bigquery.datasets.json

```diff
@@ -1,13 +1,4 @@
 [
-  {
-    "kind": "bigquery#dataset",
-    "id": "k8s-artifacts-prod:gcs_logs",
-    "datasetReference": {
-      "datasetId": "gcs_logs",
-      "projectId": "k8s-artifacts-prod"
-    },
-    "location": "US"
-  },
```

Comment:
This was me, https://github.com/kubernetes/k8s.io/issues/904#issuecomment-920448949